azurelinuxagent/distro/suse/rdma.py

```diff
@@ -35,10 +35,13 @@ def install_driver(self):
             logger.error(error_msg)
             return
         zypper_install = 'zypper -n in'
+        zypper_remove = 'zypper -n rm'
         zypper_search = 'zypper se -s'
```

Comment:
Trailing %s in command set up makes sense. Having played around a bit with refactoring the whole method and ending up in a tangled mess I see no advantage to have the commands as class or instance members, thus I think we should leave them as they are with the added "%s"